swimlane/core/fields/base/field.py

```diff
@@ -100,7 +100,7 @@ def cast_to_bulk_modify(self, value):
 
     def validate_value(self, value):
         """Validate value is an acceptable type during set_python operation"""
-        if self.readonly:
+        if self.readonly and not self._swimlane._write_to_read_only:
```

Comment:
I do not see where this value is set.

Reply:
It is set when the Swimlane client is initialized here: https://github.com/mbutlerswim/swimlane-python/blob/write-read-only/swimlane/core/client.py#L108

Reply:
But it is not a client nor does it inherit from it, so self will not be populated for field class

Reply:
The Field class inherits the SwimlaneResolver class which applies the swimlane client in the property self._swimlane.  So the _write_to_read_only property is available to the Field class at self._swimlane._write_to_read_only.

Reply:
I have also added a test now as well.